### DISCUSSION TITLE
Billing Inquiry for Additional Team Member

### DISCUSSION THREAD

**Discussion by vijayideya** [community/community#126534] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I would like to raise a ticket regarding my last month's billing, which amounted to $116 for the GitHub Team Plan. Typically, a team of 25 members costs around $100, but I noticed that my bill exceeded this amount. I require a detailed breakdown of the charges to understand the discrepancy, specifically including any tax information. This will help me better comprehend the additional charges and ensure accurate billing going forward.
Now 
I recently added an extra member to my GitHub Team, increasing the total number to 26. As a result, GitHub is now requesting an additional payment of $3.22. I need a detailed explanation of how this billing is calculated and would appreciate a breakdown of the charges. Please provide all relevant particulars.

**Comment by ebndev** [comment]
Hi @vijayideya,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account and billing questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them!

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
